client/data/settings/actions.js

```diff
@@ -64,3 +71,10 @@ export function* saveSettings() {
 		yield updateIsSavingSettings( false );
 	}
 }
+
+export function updateDigitalWalletsSectionsToShowOn( sections ) {
```

Comment:
What do you think about calling this (and the hook) just `updateDigitalWalletsSections` (without the "ToShowOn" part)?
That way it matches more closely the attribute name.
Might be worth mentioning that the attribute is currently called `woocommerce_payments_payment_request_button_locations` - maybe calling it "locations" (instead of "sections") would help once we start using the actual attributes from the gateway?

Reply:
I have no strong preferences on names. I used "section" because of the [existing code here](https://github.com/Automattic/woocommerce-payments/blob/6a9f672f0e1e6f1ca59beda54cf4d49cd317726e/client/settings/digital-wallets/index.js#L35-L39).

I will update them to "location" since it's preferred.

Reply:
That makes sense, thanks for pointing it out! :)

Reply:
Updated `sections` to `locations`. As well as renamed enum files and replaces all things.